### DISCUSSION TITLE
Incorrect link on home page

### DISCUSSION THREAD

**Discussion by raybaxter** [community/community#11685] [Other Feature Feedback, Questions, & Ideas] [answered]
[Home Page](https://cs.github.com/) shows example searches.

This one [owner:tpope extension:vim](https://cs.github.com/?q=owner%3Atpope%20extension%3Avim) is broken.

> Unrecognized qualifier. Looking for a file extension? Try using the path qualifier

**Comment by look** [comment, marked as answer]
Thanks for the report @raybaxter. This has been fixed.
